apps/studio/lib/constants/index.ts

```diff
@@ -80,6 +80,9 @@ export const LOCAL_STORAGE_KEYS = {
   AUTH_SMTP_CHANGES_WARNING: 'auth-smtp-changes-warning-dismissed',
 
   AUTH_USERS_COLUMNS_CONFIGURATION: (ref: string) => `supabase-auth-users-columns-${ref}`,
+
+  // api keys view switcher for new and legacy api keys
+  API_KEYS_VIEW: 'supabase-api-keys-view',
```

Comment:
needs to include project ref - otherwise it'll apply for _all_ projects for the user

Reply:
that's the idea...

Reply:
this isn't the feature preview no? this is this tab here?
![Uploading image.png…]()

Reply:
just ignore it
we don't know how we want it to work yet

Reply:
ah fine ill do it

Reply:
i can do this too no worries